src/_P017_PN532.ino

```diff
@@ -339,17 +328,10 @@ bool P017_handle_timer_in(struct EventStruct *event)
         tempcounter++;
 
         if (loglevelActiveFor(LOG_LEVEL_INFO)) {
-          String log = F("PN532: ");
-
-          if (new_key) {
-            log += F("New Tag: ");
-          } else {
-            log += F("Old Tag: ");
-          }
-          log += key;
-          log += ' ';
-          log += tempcounter;
-          addLogMove(LOG_LEVEL_INFO, log);
+          addLog(LOG_LEVEL_INFO, strformat(F("PN532: %s Tag: %d %u"),
+                                           FsP(new_key ? F("New") : F("Old")),
```

Comment:
What is this `FsP` ?

Reply:
`FsP` is a macro that converts the Flash string to a `String(X).c_str()` for ESP8266 and just the plain Flash string (pointer) for ESP32.

Reply:
cool :)

Reply:
Introduced ~4 months ago by moi 😉